### ISSUE TITLE
CRAN package has been orphaned. Any chance you can update so it remains viable on CRAN

### ISSUE THREAD

**Issue by dbetebenner** [renozao/doRNG#16]
Hello, 

A package I develop uses both doRNG and rngTools. I was just informed that that those packages have been orphaned and that CRAN doesn't allow dependencies to orphaned or archived packages. 

Are you abandoning the packages. Any chance you can update to keep them viable on CRAN.

Thanks

**Comment by tylermorganwall** [comment]
Yes, I am also interested in a resolution to this. I have a paper I'm currently revising that references doRNG and I'd rather not have to restructure both my package and the paper to remove the reference.

**Comment by renozao** [comment]
Apologies for the inconvenience. I'll try get those back up ASAP.

**Comment by dbetebenner** [comment]
That's fantastic news. Thank you! I've been given a February 3rd deadline to get things in line on my side (see email below). Can you keep us updated on your progress.

```
After years of unresponsiveness from the maintainer, these packages have 
finally been orphaned and are in danger of being archived. Under the 
CRAN policy, packages are no longer allowed to require them: this applies to

Counterfactual CovSelHigh FrechForest Mediana NMF RSNPset SGP SparseMDC 
WebGestaltR bdots bestNormalize bisque blockmodeling clustcurv condSURV 
fabMix fuzzyforest gbts groupedSurv lass0 momentuHMM mplot skpr survidm 
telefit ubiquity varclust

Your choices are

- to remove all dependence on the orphaned package
- to change to Suggests and use conditionally
- to take over maintainership

Please resolve by Feb 3 to safely retain your package on CRAN.
```

**Comment by petersonR** [comment]
I'm in the same boat, having received the same email from CRAN. Thank you very much for looking into this and trying to resolve it! I would be especially grateful for a progress update on or before Jan 31 so I know whether I'll need to restructure my package.

**Comment by sjwilczynski** [comment]
Same issue for me looking for updates

**Comment by sestelo** [comment]
Same issue for me! Thanks for the update!!

**Comment by dbetebenner** [comment]
@renozao   Not to pester you, but do you have any updates on getting your packages `rngTools` and `doRNG` un-orphaned on CRAN?  We all have a Feb. 3 deadline to get this resolved or our packages will be pulled from CRAN.

**Comment by HenrikBengtsson** [comment]
FYI, un-ORPHANED rngtools 1.5 is now on CRAN.

@renozao, do you know the reason for why doRNG was also ORPHANED?  I don't see any critical check issues.  Was it because of 'rngtools' and when you didn't respond to those requests, did they apply ORPHANED to all of your packages?  If the latter, I think this is useful to know for the community.  (...and it'll be interesting to see if they then un-ORPHANED it automatically)

**Comment by renozao** [comment]
@HenrikBengtsson beat me on the news.
These packages got ORPHANED because they both depend on `pkgmaker`, which got ORPHANED because of some left-over files it was leaving in the check directory.
I submitted an update for it that now does proper cleanup after the culprit examples.
An update for `doRNG` is currently under review and should be available over the next 24h.

**Comment by HenrikBengtsson** [comment]
> These packages got ORPHANED because they both depend on pkgmaker, ...

I see. To me it sounds like they ORPHANED _all_ of your packages when they flipped the switch because:
```
Package: doRNG
...
Author: Renaud Gaujoux [aut, cre]
Maintainer: ORPHANED
Repository: CRAN
Date/Publication: 2018-06-22 23:04:58
X-CRAN-Original-Maintainer: Renaud Gaujoux <renozao@protonmail.com>
X-CRAN-Comment: Orphaned on 2020-01-20 as maintainer was unresponsive.
```
It kind of makes sense; if they think you disappeared for good, then they do this.

PS. I just ran through my doFuture tests with your doRNG 1.8 and got all OKs.

**Comment by renozao** [comment]
Great news.
I had to resubmit to CRAN to fix minor things that came up with the submission.
Final version to be up is expected to be 1.8.2

**Comment by renozao** [comment]
Hi all.
New version has been deployed on CRAN.
Not sure if you need to do anything on your side, maybe just check with CRAN about this.

**Comment by renozao** [comment]
Please let me know if anybody still has problems related to this.
